src/sqlfluff/core/parser/segments/base.py

```diff
@@ -260,7 +260,7 @@ def raw_segments_upper(self) -> Optional[str]:
         # return [seg.raw_upper for seg in self.raw_segments]
 
     @cached_property
-    def templated(self) -> bool:
+    def is_templated(self) -> bool:
```

Comment:
I added this property recently. Afterwards, I realized that, for consistency with other properties of `BaseSegment` (e.g. `is_whitespace`), I should've named it with an `is_` prefix.